autometa/common/external/bedtools.py

```diff
@@ -90,13 +96,18 @@ def parse(bed, out=None):
 
     Raises
     -------
+    ValueError
+        `out` incorrectly formatted to be read as pandas DataFrame.
     FileNotFoundError
         `bed` does not exist
 
     """
-    if out and os.path.exists(out):
-        cols = ['contig','coverage']
-        return pd.read_csv(out, sep='\t', usecols=cols, index_col='contig')
+    if out and os.path.exists(out) and not os.stat(out).st_size == 0:
+        try:
+            cols = ['contig','coverage']
+            return pd.read_csv(out, sep='\t', usecols=cols, index_col='contig')
+        except ValueError as err:
+            raise ValueError(f'InvalidTableFormat: {out}')
     if not os.path.exists(bed):
         raise FileNotFoundError(bed)
     names = ['contig','depth','bases','length','breadth']
```

Comment:
See Issue #18, point 4. Perhaps `depth_fraction` is more descriptive?

Reply:
I'll switch the column name to `depth_fraction`.